### PR TITLE
Latest WIP, switch api -> trpc, some styling

### PR DESCRIPTION
**Pull request by RnZ3**
still WIP

### CODE REVIEW

File: src/pages/round/[...number].tsx [hunk 1]
```diff
@@ -75,57 +64,80 @@ export default function Round() {
...
-
-  const { data: bribedata, isLoading } = useQuery(["bribedata", Url], () =>
-    fetchRoundData(Url)
-  );
-
-  if (isLoading) {
-    return <Center>Loading ... </Center>;
-  }
+  console.log(bribeData?.header);
+  console.log(bribeData?.bribelist);
```
Comment (onkeljoe) [inline src/pages/round/[...number].tsx:70]: maybe you should check for valid bribeData and display an error message, if not availabe, like
`if (!bribeData) return (<>Error: Data not available, please come back later.</>)` - with some formatting you like. 
Then you do not need the questionmark-operator after each occurance of bribeData(?).
Just a suggestion for your next improvement.

**Review by onkeljoe — APPROVED** [review]

File: src/pages/round/[...number].tsx [hunk 2]
```diff
@@ -75,57 +64,80 @@ export default function Round() {
...
-        <Box>
-          <pre>{bribedata?.version}</pre>
-        </Box>
-      </Center>
+      <Box>
+        <Center>
+          <Text fontSize="4xl">{bribeData?.header.roundName}</Text>
+        </Center>
+        <Center>
+          <Text fontSize="1.4rem">
```
Comment (mobiusTripper-crypto) [inline src/pages/round/[...number].tsx:79]: This is good for now, but on the todo list should be bake this style into the theme

File: src/pages/round/[...number].tsx [hunk 3]
```diff
@@ -75,57 +64,80 @@ export default function Round() {
...
+          </Text>
+        </Center>
+      </Box>
+
       {display === "cards" ? (
         <Center>
           <Wrap justify="center" padding="23px" margin="23px" spacing="50px">
-            {bribedata?.bribedata?.map((bribe: any, i: number) => (
+            {bribeData?.bribelist?.map((bribe: any, i: number) => (
               <Box
```
Comment (mobiusTripper-crypto) [inline src/pages/round/[...number].tsx:103]: This is also fine for now, but seems like we could leverage the chakra <Card> for this box (and put all the styles in the theme as a variant)

**Review by mobiusTripper-crypto — APPROVED** [review]

File: src/pages/round/[...number].tsx (hunk 3 above)
Comment (mobiusTripper-crypto) [inline src/pages/round/[...number].tsx:103]: (i can't get my comment to go in the exact spot i want, but this is in reference to the "card")

This is also fine for now, but seems like we could leverage the chakra for this box (and put all the styles in the theme as a variant)

**Review by mobiusTripper-crypto — APPROVED** [review]

**Pull request merged**